softmax/Dockerfile

```
@@ -7,15 +7,33 @@ COPY . /workspace/metta
 
 WORKDIR /workspace/metta
 
-# Install system dependencies
-RUN bash ./devops/tools/install-system.sh
+# Install bootstrap dependencies (bazel, nimby, nim, git, g++) before uv sync
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    python3 \
+    python3-pip \
+    python3-venv \
+    curl \
+    git \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install uv if not present
+RUN curl -LsSf https://astral.sh/uv/install.sh | sh || true
+ENV PATH="/root/.local/bin:$PATH"
```

iirc these env changes dont persist out of build stage into the runtime stage, but I could be wrong. we should be careful becuase runtime will want to `metta install` again in case the code has changed since last docker img build

---

uv is copied to /bin/ which is in default PATH, so it persists. Added `uv --version` check to fail fast if missing.